python/ray/tests/test_actor.py

```diff
@@ -1393,6 +1394,27 @@ def get_actor(actor):
     assert origin == remote
 
 
+@pytest.mark.skipif(
+    is_python_version_older_than(3, 9), reason="GenericAlias is new in Python 3.9"
```

Comment:
NIt: You can directly do `sys.version_info < (3, 9)`. A new function seems an overkill.

Reply:
Updated https://github.com/ray-project/ray/pull/43117/commits/5c472dcf6ee97644e78daa521c0f91f8fce9e759.